src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -26,6 +26,8 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
+import java.util.Set;
+import java.util.HashSet;
 
```

Comment:
Make sure the import are in alphabetic order - from the java style guide (https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)